.changeset/thick-fans-scream.md

```diff
@@ -0,0 +1,11 @@
+---
+"@khanacademy/kas": minor
+"@khanacademy/kmath": minor
+"@khanacademy/math-input": minor
+"@khanacademy/perseus-error": minor
+"@khanacademy/perseus-linter": minor
+"@khanacademy/pure-markdown": minor
+"@khanacademy/simple-markdown": minor
```

Comment:
Shouldn't these be major changes as well?

Reply:
They haven't hit 1.0.  A minor bump in pre-1.0 packages is equivalent to a major bump in post-1.0 packages according to semver.